src/main/java/service/ReportService.java

```diff
@@ -0,0 +1,6 @@
+package service;
+
+public interface ReportService {
+
+    String getReportStringForWriting();
```

Comment:
The report may not only be for the writing, so don't get attached to such names
```suggestion
    String getReport();
```